src/main/java/dev/lsegal/jenkins/codebuilder/CodeBuilderCloud.java

```diff
@@ -370,14 +372,20 @@ public synchronized AWSCodeBuild getClient() {
    */
   private long numStillProvisioning() {
     return jenkins().getNodes().stream().filter(CodeBuilderAgent.class::isInstance).map(CodeBuilderAgent.class::cast)
-        .filter(a -> a.getLauncher().isLaunchSupported()).count();
+        .filter(a -> !a.getLauncher().isLaunchSupported()).count();

```

Comment:
Is there a reason this value is negated? Was this broken?

Reply:
We have taken this PR (and some others) into a fork, to be able to use them on our instance. We had to roll back this particular line though (as you mentioned, it is not broken on master). See https://github.com/cvent/codebuilder-cloud-plugin/commit/1dfd98fc49f05e24c9cd9be9626e95192011433a for the change we made to roll this line back (and some other comments that helped to track the logic around the launch successes)